src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -1046,14 +1065,14 @@ sourceFile, getSourceStorageAccountPrimarySharedKey()))
     datasetId = null;
 
     // Make sure that any failure in tearing down is presented as a test failure
-    blobIOTestUtility.deleteContainers();
+    azureBlobIOTestUtility.teardown();
```

Comment:
I don't think this is necessary as we call this teardown method as part of test teardown.  (Ditto throughout this test suite.)

Reply:
I think that was there to get the error to show up as a test failure.  I look into handling this in a followup if that's ok